### DISCUSSION TITLE
Crashlytics installation

### DISCUSSION THREAD

**Discussion by stacy-klyashtorna** [invertase/react-native-firebase#5691] [Q&A]
Hello everyone !

I tried to to install `@react-native-firebase/crashlytics` but I got error in xCode. Maybe something had this problem before and can help to solve it ? 
<img width="1527" alt="Screenshot 2021-09-03 at 11 43 27" src="https://user-images.githubusercontent.com/85610835/131977332-138251a4-5306-49d6-84d5-458c2b9cad37.png">
<img width="1374" alt="Screenshot 2021-09-03 at 11 44 02" src="https://user-images.githubusercontent.com/85610835/131977367-4314b033-3816-4f31-aa0f-676141a2d227.png">

My app is using expo bare workflow *

**Comment by mikehardy** [comment]
Hi there!
As a rule, do not post images when asking for help, they work terribly for people on small devices like phones etc.
As a rule, please post all relevant information when asking for help, this is for instance more of an issue, not a discussion, and our issue template asks for relevant information like your Podfile etc which would help here.

The problem you have is also with RNOpenSettings, it's not specific to react-native-firebase, so what you have is a general iOS compilation issue, not a react-native-firebase issue, and posting here will get you limited help, vs perhaps stack overflow, where there is more information about the problem.

I also had a similar problem, and stack overflow is where I got my answers. You need to constrain the architectures allowed for your build.

We do the same here for our build to work successfully, example:

https://github.com/invertase/react-native-firebase/blob/90b4d7021aff7d5d28140b57eb69283b04712257/tests/ios/Podfile#L54-L55

Then during the build itself (so it works on Apple Sillicon or Intel Macs) https://github.com/invertase/react-native-firebase/blob/90b4d7021aff7d5d28140b57eb69283b04712257/tests/package.json#L60
